inductiva/utils/templates.py

```diff
@@ -10,24 +12,33 @@
 
 
 def replace_params_in_template(
-    templates_dir: str,
-    template_filename: str,
+    template_path: str,
     params: Dict,
     output_file_path: str,
+    remove_template: bool = False,
 ) -> None:
     """Replaces parameters in a template file."""
 
-    environment = Environment(loader=FileSystemLoader(templates_dir))
+    template_path = Path(template_path)
+
+    template_dir = template_path.parent
+    template_filename = template_path.name
+
+    environment = Environment(loader=FileSystemLoader(template_dir))
     template = environment.get_template(template_filename)
     stream = template.stream(**params)
     stream.dump(output_file_path)
 
+    if remove_template:
```

Comment:
Can you clarify if this removes the template files from the new output_path? As it is, it just seems to remove from the local folder where we are pointing for the templates.

Reply:
This function takes as input a template file and produces an output file. This if condition determines whether to remove the template file after creating the output file.

This is particularly useful when we copy a directory with one or more template files to another location that serves as the input directory for a simulation, and then replace parameters in those template files. It can be used to immediately remove the template files after their conversion is completed.